lib/eex/lib/eex/compiler.ex

```diff
@@ -27,7 +28,7 @@ defmodule EEx.Compiler do
           quoted: [],
           start_line: nil,
           start_column: nil,
-          parser_options: Code.get_compiler_option(:parser_options)
+          parser_options: Code.get_compiler_option(:parser_options) ++ parser_options
```

Comment:
```suggestion
          parser_options: Keyword.merge(Code.get_compiler_option(:parser_options), parser_options)
```

Reply:
I used `++` because it's used where the options are passed into `Code.string_to_quoted` -> https://github.com/elixir-lang/elixir/blob/24f8db1a40336e3cc0c517e048b77e75ff8123ad/lib/eex/lib/eex/compiler.ex#L59
Out of curiosity, is there a reason to prefer `Keyword.merge`?

Reply:
You can use `++` but the options that come first win, which is likely not what you want here. There is another option, which is to do:

    parser_options = opts[:parser_options] || Code.get_compiler_options(:parser_options)

Perhaps this is even preferred as the intent is clear?

Reply:
You're right, that's not what we want here.
I'll use `parser_options = opts[:parser_options] || Code.get_compiler_options(:parser_options)` as it's definitely more clear about what we intend to do.

Reply:
@josevalim since doing it this way will entirely override anything set in `Code.get_compiler_options(:parser_options)` do we want an additional test that checks `get_compiler_options` options are no longer used?

Also, isn't this technically breaking, assuming someone relies on `get_compiler_options` for their EEx templates?

Reply:
I can see someone being surprised that their global configuration is not taking effect but I can also see someone surprised their options are mixed with global ones. So I think the full override behavior is clearer and folks can merge upstream if they desire.